templates/MetadataGenerator/MD_DataIdentification.xml.twig

```diff
@@ -72,6 +72,13 @@
             }
         %}
     </gmd:descriptiveKeywords>
+    <gmd:resourceConstraints xlink:title="Liability and Warranty">
+        <gmd:MD_LegalConstraints>
+            <gmd:otherConstraints>
+                <gco:CharacterString>All materials on this website are made available to GRIIDC and in turn to you “as-is.” Content may only be submitted by an individual who represents and warrants that s/he has sufficient rights to be able to make the content available under a CC0 waiver. There is no warranty (expressed or implied) to these materials, their title, accuracy, non-infringement of third party rights, or fitness for any particular purpose, including the performance or results you may obtain from their use. Use these materials at your own risk.  Under no circumstances shall GRIIDC be liable for any direct, incidental, special, consequential, indirect, or punitive damages that result from the use or the inability to use either this website or the materials available via this website. If you are dissatisfied with any website feature, content, or terms of use, your sole and exclusive remedy is to discontinue use.</gco:CharacterString>
```

Comment:
Maybe not use the funky quotes, but replace them with plain double quotes. 
Text: `“as-is.”`

Reply:
Copy pasted from JIRA, thats why the funky double quotes